### DISCUSSION TITLE
Tooltips for icons

### DISCUSSION THREAD

**Discussion by JasonCSH9566** [elementor/elementor#23635] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

It's very frustrating that when adding an icon/icon box widget there is not method of adding a tooltip on hover.  Sometimes an icon is better without text next to it.  Tooltips are a fundamental part of accessibility.

### Describe the solution you'd like

It would be great and very useful if when adding an icon there is a tooltip feature under the hover tab in styling.  Where you can style the text and background of the tooltip.

### Describe alternatives you've considered

I've tried using custom css and also the hotspot widget

**Comment by 79ho3ein** [comment]
In general, Elementor should consider tooltip feature for all widgets. https://github.com/elementor/elementor/issues/16415

**Reply by JasonCSH9566** [reply]
Absolutely.  Apologies for the duplicate request.  I had searched on Elementor and assumed it was referring to that.
